patroni/scripts/wale_restore.py

```diff
@@ -50,18 +51,28 @@ def get_major_version(data_dir):
     return 0.0
 
 
+WALEConfig = namedtuple('WALEConfig',
+                        'dir,threshold_mb,threshold_pct,iam_string,cmd')
```

Comment:
Once you started refactoring, may be it make sense to remove `dir` and `iam_string` from `WALEConfig`?
They are not used anywhere.

Everything else seems fine.